docs/docs/gatsby-provision.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This doc shouldn't be added to the `/docs/docs` directory but be grouped into one of the child folders. What type of guide is this? How-To, Reference?

Reply:
Yep! This was a temporary home for it while we discussed where the best place for it to live would be.

I see this more as a reference than a how-to, and most akin to a configuration file, so I've opted to place it under `/reference/config-files/`.